.travis.yml

```diff
@@ -1,17 +1,17 @@
 sudo: required
-dist: trusty
+dist: xenial
 
 addons:
   apt:
     sources:
-      - chef-current-trusty
+      - chef-stable-xenial
     packages:
       - chefdk
       - docker-ce
 
 services: docker
 
-install: chef exec bundle
+install: CHEF_LICENSE="accept" chef exec bundle
```

Comment:
Ah hah! Nice, this is how you do it. I did read the doc, hoping to provide something a bit more specific but I couldn't get through the density of it. Thanks!